src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -26,6 +26,10 @@ const QString kRequestPath = QStringLiteral("/ws/2/recording/");
 
 const QByteArray kUserAgentRawHeaderKey = "User-Agent";
 
+// MusicBrainz allows only a single request per second on average
+// to avoid rate limiting. Otherwise requests fail with 503.
+constexpr Duration kMinDurationBetweenRequests = Duration::fromMillis(1000);
```

Comment:
```suggestion
// MusicBrainz allows only a single request per second on average
// to avoid rate limiting. 
// See: https://musicbrainz.org/doc/MusicBrainz_API/Rate_Limiting
constexpr Duration kMinDurationBetweenRequests = Duration::fromMillis(1000);
```